partcad-cli/src/partcad_cli/cli.py

```
@@ -129,92 +108,21 @@ def main():
-        elif args.command == "install" or args.command == "update":
-            with pc.logging.Process("Install", "this"):
-                cli_install(args, ctx)
-
-        elif args.command == "list":
-            with pc.logging.Process("List", "this"):
-                cli_list(args, ctx)
-
-        elif args.command == "list-all":
+        if args.command == "list-all":
             with pc.logging.Process("ListAll", "this"):
                 cli_list_sketches(args, ctx)
```

Ruff (0.8.0)

`cli_list_sketches` may be undefined, or defined from star imports

(F405)